pkg/scalers/azure_app_insights_scaler.go

```diff
@@ -24,13 +24,23 @@ const (
 	azureAppInsightsMetricAggregationTypeName     = "metricAggregationType"
 	azureAppInsightsMetricFilterName              = "metricFilter"
 	azureAppInsightsTenantIDName                  = "tenantId"
+	azureAppInsightsIgnoreNullValues              = "ignoreNullValues"
+)
+
+var (
+	azureAppInsightsDefaultIgnoreNullValues = true
```

Comment:
I think that this changes the current behavior. Currently, we raise an error instead of returning 0. We should introduce this disabled as default to not break any behaviour

Reply:
Changed to false